environments/adapters/ollama.py

```
@@ -1,12 +1,14 @@
 from __future__ import annotations
 
 import json
+import shlex
 import shutil
 from pathlib import Path
 from typing import Mapping
```

Ruff (0.16.1)

[warning] Import from `collections.abc` instead: `Mapping`

Import from `collections.abc`

(UP035)